### PR TITLE
Updated homepage!

### PR DESCRIPTION
**Pull request by Deepcryptodive**
- Added flavicon (so a little ghost is shown in the browser tab)
- Updated homepage text & layout
- Updated team page
- Updated FAQ

Unchanged: top header which does not link correctly to the roadmap & FAQ. This still needs fixing.

### CODE REVIEW

File: src/components/sections/FeaturesTiles.js [hunk 1]
```diff
@@ -122,80 +126,20 @@ class FeaturesTiles extends React.Component {
                   </div>
                   <div className="features-tiles-item-content">
                     <h4 className="mt-0 mb-8 reveal-from-top" data-reveal-container=".tiles-item" data-reveal-delay="300">
-                       Get rewarded
+                       Watch interest accrue
                     </h4>
                     <p className="m-0 text-sm reveal-from-top" data-reveal-container=".tiles-item" data-reveal-delay="400">
-                    Everyone gets their principle back, but the savvy savers who made every payment, split the interest. Hit your goals to get higher interest.
+                    Pooled funds are deposited in DeFi savings accounts, generating higher interest rates than in traditional finance. Annual rates of over 7% are possible - shared between all winners.
```
Comment (RachBLondon) [inline src/components/sections/FeaturesTiles.js:132]:
```suggestion
                    Pooled funds are deposited in DeFi savings accounts, generating higher interest rates than in traditional finance. Annual rates of over 7%* are possible - shared between all winners. *Returns vary and are based on the interest rate provided by <a href="https://aave.com/" target="_blank" rel="noopener noreferrer">Aave</a>.
```

File: src/components/sections/HeroSplit.js [hunk 2]
```diff
@@ -71,16 +71,16 @@ class HeroSplit extends React.Component {
...
-                  The new addictive way to save. Our incentivised pool rewards regular savers with higher interest rates. Start building the financial habits you deserve today.
+                  The new addictive way to save. Our incetivized pools rewards regular savers with higher interest rates.<br/>Start building the financial habits you deserve.
                   {/* Join our pool, hit your goals and get rewarded. Interest rates of over 7% are possible. The more people drop out, the higher your returns. */}
                   </p>
                   <div className="reveal-from-top" data-reveal-delay="450">
-                   
-                    <Button tag="a" color="primary"  wideMobile>
-                      <a className="cta-button" href="https://discord.com/invite/AWvcTFP">Join our discord</a>
+
+                    <Button tag="a" color="primary" wide="true" wideMobile="true">
```
Comment (RachBLondon) [inline src/components/sections/HeroSplit.js:82]: Have you tested this attribute out locally? wide="true"

File: src/components/sections/HeroSplit.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ class HeroSplit extends React.Component {
 // inline style
 const inlineCss = {
   alignItems: 'flex-start',
-  minHeight: '492px'
+  minHeight: '100px'
```
Comment (RachBLondon) [inline src/components/sections/HeroSplit.js:99]: I think this styling came with the theme - we should leave unless there is a specific reason to change it.

**Review by RachBLondon — APPROVED** [review]

File: src/components/sections/HeroSplit.js (hunk 2 above)
Comment (Deepcryptodive) [reply src/components/sections/HeroSplit.js:82]: Yes, it makes the button slightly bigger

File: src/components/sections/FeaturesTiles.js (hunk 1 above)
Comment (Deepcryptodive) [reply src/components/sections/FeaturesTiles.js:132]: Good idea to add the asterix!
Will commit for now, but we might have to move the text a bit down if it messes with the layout.

File: src/components/sections/HeroSplit.js (hunk 3 above)
Comment (Deepcryptodive) [reply src/components/sections/HeroSplit.js:99]: It was just to reduce whitespace a bit. I don't think it breaks any of the stylin - but good to keep an eye on this.

**Pull request merged**